CODING_GUIDELINES.md

```diff
@@ -43,24 +43,24 @@ UAT should be revisited at each stage of the product development:
 
 ![acceptance-tests.png](./docs/assets/acceptance-tests.png)
 
-### Why Acceptance Testing?
+### Why Acceptance Testing
 
-- Automated acceptance tests catch serious problems that unit or component test suites could never catch.
-- Automated acceptance tests deliver business value the users are expecting as they test user scenarios.
-- Automated acceptance tests executed and passed on every build help improve the software delivery process.
-- Testers, developers, and customers need to work closely to create suitable automated acceptance test suites.
++ Automated acceptance tests catch serious problems that unit or component test suites could never catch.
```

Comment:
nit: would be good to make all bullet points -, + or * instead of each file having their own. 

a thought, no need to do this in this PR

Reply:
True, I can specify `"MD004": {"style":  "asterisk"},` in the markdownlint config if you want.

Reply:
I like the idea. Would be nice to be more consistent.

Reply:
if you want to tackle it, please, otherwise we can open an issue. Its not a blocker on my side